### ISSUE TITLE
Version 3.6.0 is not available for installation.

### ISSUE THREAD

**Issue by lunxer** [SBFspot/sbfspot-config#14]
Trying to get update to newest sbfspot to get MQTT going.

`curl -s https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/sbfspot-config | sudo bash -s 3.6.0`

```
Request to install version 3.6.0
Version 3.6.0 is not available for installation.
Select one of 3.5.0 3.5.1 3.5.2
```

**Comment by sbf-** [comment]
I know, I'm still working on MQTT integration.
It takes a bit more time than expected, but it's almost finished (I hope)

**Comment by sbf-** [comment]
Fixed in V1.1.0
